internal/bots/telegram/handlers/summarize/channel_post_test.go

```diff
@@ -15,9 +15,12 @@ var h *Handler
 func TestMain(m *testing.M) {
 	logger := logger.NewLogger(logrus.DebugLevel, "insights-bot", "", nil)
 	config := configs.NewConfig()()
-	openaiClient := thirdparty.NewOpenAIClient()(thirdparty.NewOpenAIClientParam{
+	openaiClient, err := thirdparty.NewOpenAIClient()(thirdparty.NewOpenAIClientParam{
 		Config: config,
 	})
+	if err != nil {
+		os.Exit(1)
```

Comment:
How about we use `panic(err)` to replace `os.Exit(1)` since the user should be able to know what happened during development and within CI/CD?